src/engine/controls/loopingcontrol.cpp

```diff
@@ -406,8 +417,14 @@ double LoopingControl::nextTrigger(bool reverse,
                 }
                 if (movedOut) {
                     *pTarget = seekInsideAdjustedLoop(currentSample,
-                            loopSamples.start, loopSamples.start, loopSamples.end);
+                            loopSamples.start,
+                            loopSamples.start,
+                            loopSamples.end);
                 }
+                break;
+            case LoopSeekMode::None:
+                // Nothing to do here
```

Comment:
Can this ever happen? If not a DEBUG_ASSERT might be useful.

Reply:
Yes, i added a comment.